libcaliptra/inc/caliptra_types.h

```diff
@@ -63,3 +63,117 @@ struct caliptra_stash_measurement_resp {
     struct caliptra_completion cpl;
     uint32_t                   dpe_result;
 };
+
+struct caliptra_get_idev_csr_resp {
+    struct caliptra_completion cpl;
+    uint32_t                   data_size;
+    uint8_t                    data[1024];
+};
+
+struct caliptra_get_ldev_cert_resp {
+    struct caliptra_completion cpl;
+    uint32_t                   data_size;
+    uint8_t                    data[1024];
+};
+
+// The below fields are placeholders to set up the baseline
+// required for communication of DPE commands to Caliptra
+// firmware.
+
+#define DPE_DATA_MAX 512
+#define DPE_MAGIC    0x44504543 // "DPEC"
+
+struct dpe_cmd {
+    uint32_t magic;
+    uint32_t cmd_id;
+    uint32_t profile;
+};
+
+struct dpe_response {
+    uint32_t magic;
+    uint32_t status;
+    uint32_t profile;
+};
```

Comment:
nit: I would recommend naming these something like `dpe_cmd_hdr` and `dpe_resp_hdr`

Reply:
Can do.